app/controllers/delivery_options_controller.rb

```diff
@@ -0,0 +1,62 @@
+class DeliveryOptionsController < ApplicationController
+  before_action :set_seller
+  before_action :set_seller_delivery_options, only: [:update, :destroy]
+
+  # GET /sellers/:seller_id/delivery_options
+  def index
+    delivery_options = @seller.delivery_options.map do |delivery_option|
+      delivery_type = delivery_option.delivery_type
+      json = delivery_option.as_json
+      json['delivery_type'] = delivery_type.as_json
```

Comment:
Mind testing this in a unit test?

Reply:
Yeah I was having trouble finding an example of a unit test of a has_* association - any advice?

Reply:
https://github.com/sendchinatownlove/ruby/pull/178/files#r460165708